modules/account-lib/test/unit/coin/dot/keypair.ts

```diff
@@ -0,0 +1,101 @@
+import should from 'should';
+import { Dot } from '../../../../src';
+import * as DotResources from '../../../resources/dot';
+
+describe('Dot KeyPair', () => {
+  const defaultSeed = { seed: Buffer.alloc(32) };
+
+  const {
+    accounts: { account1, account2, account3, default: defaultAccount },
+  } = DotResources;
+
+  describe('Keypair creation', () => {
+    it('initial state', () => {
+      const keyPair = new Dot.KeyPair();
+      const decodedKeys = Dot.Utils.default.decodeDotKeyringPair(keyPair.getKeys());
```

Comment:
`getKeys()` should already return `prv` and `pub`

Reply:
https://github.com/BitGo/BitGoJS/pull/1658#:~:text=feat(account-lib)%3A%20dot%20keypair%20fix

restored